packages/ember-htmlbars/lib/keywords/mut.js

```diff
@@ -63,11 +63,32 @@ let MutStream = ProxyStream.extend({
   // my-child.js
   export default Component.extend({
     click() {
-      this.attrs.childClickCount.update(this.attrs.childClickCount.value + 1);
+      this.childClickCount.update(this.childClickCount.value + 1);
     }
   });
   ```
 
+  Additionally, the `mut` helper can be combined with the `action` helper to more
+  easily mutable a value. This is particularlly helpful if you're following the
+  data down, actions up philosophy. For example:
+
+  ```handlebars
+  {{my-child childClickCount=totalClicks click-count-change=(action (mut "totalClicks"))}}
+  ```
+
+  The child `Component` would invoke the action with the new click value:
+
+  ```javascript
+  // my-child.js
+  export default Component.extend({
+    click() {
+      this['click-count-change'](this.childClickCount + 1);
```

Comment:
@locks Dang! I almost did that, but it wasn't using `get` before so I thought I shouldn't. That'll teach me to go against my instincts.